hivemq-edge/src/frontend/src/components/rjsf/BatchSubscription/components/ColumnMatcherStep.tsx

```diff
@@ -0,0 +1,7 @@
+import { FC } from 'react'
+
+const ColumnMatcherStep: FC = () => {
+  return <div></div>
```

Comment:
question: i guess these are placeholders?

```suggestion
  return <div></div>
```